source/draw/gpu/GpuCanvas.ooc

```diff
@@ -55,4 +55,9 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		this _target y canvas clear()
 		this _target uv canvas clear()
 	}
+	draw: override func ~mesh (image: Image, mesh: GpuMesh) {
```

Comment:
Okay, if we want to possibly be able to call this function with a `RasterImage`, fine.